# Define the pins
tank_contol:
  ros__parameters:
    left_motor_forward: 20
    left_motor_backward: 21 
    right_motor_forward: 19
    right_motor_backward: 26
    left_motor_pwm: 16
    right_motor_pwm: 13

    timeout: 2.0
    rate: 50.0
    max_speed: 2.0
    max_pwm: 255
    # The distance between two wheel(track)
    wheel_base: 1 #0.15

imu_publisher:
  ros__parameters:    
    imu_frame_id: imu_link
    imu_bus: 3
    accel_offset_x: -0.450
    accel_offset_y: -0.392
    accel_offset_z: 0.291
    gyro_offset_x: 0.393
    gyro_offset_y: -0.073
    gyro_offset_z: 0.695
    imu_rate: 50.0
